modules/apps/dynamic-data-mapping/dynamic-data-mapping-data-provider-impl/src/test/java/com/liferay/dynamic/data/mapping/data/provider/internal/DDMDataProviderInvokerImplTest.java

```diff
@@ -254,8 +274,17 @@ public void testFetchDDMDataProviderInstance2() throws Exception {
 		DDMDataProviderInstanceService ddmDataProviderInstanceService =
 			Mockito.mock(DDMDataProviderInstanceService.class);
 
-		ddmDataProviderInvokerImpl.ddmDataProviderInstanceService =
-			ddmDataProviderInstanceService;
+		Snapshot snapshot = Mockito.mock(Snapshot.class);
+
+		Mockito.when(
+			snapshot.get()
+		).thenReturn(
+			ddmDataProviderInstanceService
+		);
+
+		ReflectionTestUtil.setFieldValue(
+			ddmDataProviderInvokerImpl,
+			"ddmDataProviderInstanceServiceSnapshot", snapshot);
```

Comment:
@devpedrotavares move the `DDMDataProviderInvokerImpl ddmDataProviderInvokerImpl =` like you did for the other two places.